pkg/core/native/native_neo.go

```diff
@@ -189,10 +191,15 @@ func (n *NEO) Initialize(ic *interop.Context) error {
 	}
 	n.mint(ic, h, big.NewInt(NEOTotalSupply))
 
-	gr := &state.GASRecord{{Index: 0, GASPerBlock: *big.NewInt(5 * GASFactor)}}
+	var index uint32 = 0
+	value := big.NewInt(5 * GASFactor)
+	key := make([]byte, 5)
+	key[0] = prefixGASPerBlock
+	binary.BigEndian.PutUint32(key[1:], index)
```

Comment:
Some helper for the key?

Reply:
United this part of code with `putBigInteger`, now we have `putGASRecord` method.